src/traces/bar/calc.js

```diff
@@ -33,7 +33,7 @@ module.exports = function calc(gd, trace) {
 
     // set position and size
     for(var i = 0; i < serieslen; i++) {
-        cd[i] = { p: pos[i], s: size[i] };
+        cd[i] = { p: pos[i], s: size[i], v: size[i] };
```

Comment:
From the 

```js
 value = cdi.s || cdi.v || cdi.y;
```

line in `sortAxisCategoriesByValue`, looks like you no longer need to set `v`.

Reply:
You're right! Nice catch :eyes: 

Removed in 2d379f7647bec909b806f56276b6f6225578bc6b